jsonschema/infer.go

```diff
@@ -8,12 +8,33 @@ package jsonschema
 
 import (
 	"fmt"
+	"log/slog"
+	"math/big"
 	"reflect"
 	"regexp"
+	"time"
 
 	"github.com/modelcontextprotocol/go-sdk/internal/util"
 )
 
+// ForOptions are options for the [For] function.
+type ForOptions struct {
+	// If IgnoreBadTypes is true, fields that can't be represented as a JSON Schema
```

Comment:
Is there any better word than 'Bad'? I'd say 'Unsupported' is better, but it's too long.

Probably Bad is best, I just wanted to note this decision.